src/bidiMapper/domains/events/EventManager.ts

```diff
@@ -210,8 +209,8 @@ export class EventManager implements IEventManager {
     eventName: Session.SubscriptionRequestEvent,
     contextId: CommonDataTypes.BrowsingContext | null
   ) {
-    // Enable network domain if user subscribed to any of network events.
-    if (eventName.startsWith(EventManager.#NETWORK_DOMAIN_PREFIX)) {
+    // Enable network domain if user subscribed to any network event.
```

Comment:
why these changes are in this PR? (looks like added by mistake)

Reply:
ooops, committed to the wrong branch